tests/fixtures/scenarios/auth-invalid/atmos.yaml

```
@@ -0,0 +1,27 @@
+  colors: false
+
+logs:
+  verbose: false
+  colors: false
+
+auth:
+  providers:
+    invalid-provider:
+      kind: invalid/provider
+      region: us-east-1
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)